pandas/tseries/offsets.py

```diff
@@ -1951,7 +1941,7 @@ class FY5253Quarter(DateOffset):
         or 14 week when needed. {1,2,3,4}
     variation : str
         {"nearest", "last"} for "LastOfMonth" or "NearestEndMonth"
-    """
+    """ % _5253doc
```

Comment:
can you use ``.format()``

Reply:
and same on all the templates (generally we want to always use the .format())

Reply:
That was the first attempt.  That syntax doesn't play nicely with the set syntax inside the docstring.  The docstring change here is tangential to the PR; I'll just revert it.